### PR TITLE
MT-010

### PR DESCRIPTION
**Pull request by GilberttValentine**

### CODE REVIEW

File: lib/auth/screens/sign-in.screen.dart [hunk 1]
```diff
@@ -101,254 +98,283 @@ class _SignInScreenState extends State<SignInScreen>
...
+                            width: 392.72727272727275,
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              children: <Widget>[
+                                Container(
+                                  padding:
+                                      const EdgeInsets.fromLTRB(0, 0, 5, 0),
+                                  child: const Text(
+                                    "Don't have an account?",
+                                    style: TextStyle(),
```
Comment (SebastianObando) [inline lib/auth/screens/sign-in.screen.dart]: Add style or delete property pls. (●'◡'●)

File: lib/auth/screens/sign-up.screen.dart [hunk 2]
```diff
@@ -1,12 +1,415 @@
...
+                          ),
+                          Row(
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              children: <Widget>[
+                                Container(
+                                  padding:
+                                      const EdgeInsets.fromLTRB(0, 0, 5, 0),
+                                  child: const Text(
+                                    'Already have account?',
+                                    style: TextStyle(),
```
Comment (SebastianObando) [inline lib/auth/screens/sign-up.screen.dart]: Same. ༼ つ ◕_◕ ༽つ

File: lib/auth/screens/sign-in.screen.dart (hunk 1 above)
Comment (GilberttValentine) [reply lib/auth/screens/sign-in.screen.dart]: Solved

File: lib/auth/screens/sign-up.screen.dart (hunk 2 above)
Comment (GilberttValentine) [reply lib/auth/screens/sign-up.screen.dart]: Solved

**Review by SebastianObando — APPROVED** [review]
Well done!

**Review by ErrorHyperion — APPROVED** [review]

**Review by CesarPava — APPROVED** [review]

**Pull request merged**